test/Feature/InsertSpec.hs

```diff
@@ -427,21 +480,21 @@ spec = do
             matchHeaders = ["Content-Range" <:> "*/*"]
           }
 
-        get "/items" `shouldRespondWith`
```

Comment:
were these tests failing after the change or is it just cleanup since it's a GET in a section about POST?

Reply:
It's just a cleanup. Didn't consider this test relevant and I would have had to update it with more `{"id": ..}` to keep it working.